openedx/features/student_account/tests/test_compose_and_send_activation_email_custom.py

```diff
@@ -0,0 +1,49 @@
+from mock import patch
+
+from django.test import RequestFactory, TestCase
+
+from student.tests.factories import RegistrationFactory, UserFactory
+
+from openedx.features.student_account.helpers import compose_and_send_activation_email_custom
+
+
+class ComposeAndSendActivationEmailTests(TestCase):
```

Comment:
I think we should have only one test file for helpers. That file can have multiple classes